components/TARGET_PSA/services/crypto/COMPONENT_SPE/TARGET_MBED_SPM/psa_crypto_srv_partition.c

```diff
@@ -23,6 +23,8 @@
  * Generated by tools/spm/generate_partition_code.py Version 1.0
  **********************************************************************************************************************/
 
+#if !defined(TARGET_TFM)
```

Comment:
If this file moved to a `TARGET_MBED_SPM` folder than this check is not necessary anymore.